src/SeleniumLibrary/utils/events/event.py

```diff
@@ -15,9 +15,30 @@
 # limitations under the License.
 
 import abc
-
+from selenium.webdriver.support.event_firing_webdriver import EventFiringWebDriver
 
 class Event:
     @abc.abstractmethod
     def trigger(self, *args, **kwargs):
         pass
+
+def _unwrap_eventfiring_elements(ef_driver,elements):
+    """ Workaround for Selenium 3 bug.
+
+        References:
+        https://github.com/SeleniumHQ/selenium/issues/7877
+        https://github.com/SeleniumHQ/selenium/pull/8348
+        https://github.com/SeleniumHQ/selenium/issues/7467
+        https://github.com/SeleniumHQ/selenium/issues/6604
+
+    """
+    if not isinstance(ef_driver,EventFiringWebDriver) or selenium_major_version() >= 4:
+        return elements
+    unwrapped_elements = ef_driver._unwrap_element_args(elements)
+    return unwrapped_elements
+
+def selenium_major_version():
```

Comment:
The distutils [LooseVersion](https://github.com/python/cpython/blob/master/Lib/distutils/version.py#L269) could be handy.

Reply:
I was trying to avoid having to require another Python package for something temporary. Right now it doesn't look like we require setuptools (which it appears LooseVersion now a part of) in either the core or the library. Although I though there might be something out there I went with this solution because of that. It does look like LooseVersion is in c-python but had not checked iron python or any other build (and don't recall which ones are supported still).

Reply:
Next version, 5.0, will support Python 3 and PyPy 3 only. No need to worry about other interpreters. But it is just an idea and your code works as well.